README.md

```diff
@@ -58,13 +58,11 @@ Build instructions
     builds wheel for by following the instructions at
     https://wiki.python.org/moin/WindowsCompilers.
 
-    Visual Studio 2019 will work for Python 3.5-3.8. For Python 3.5 support,
-    you will need to check `VC++ 2015.3 v140 toolset for desktop (x86,x64)`
-    during installation.
+    Visual Studio 2019 will work for Python 3.6-3.8.

```

Comment:
So what about 3.9? We should document the toolchain you (will) use to compile those.

Reply:
> So what about 3.9? We should document the toolchain you (will) use to compile those.

I missed that one, thanks. I used the same toolchain for everything so I amended this to say 3.9.